pkg/apis/storage/types.go

```diff
@@ -94,10 +94,7 @@ type StorageClassList struct {
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// Captures the intent to attach or detach the specified volume to/from
-// the specified node.
-//
-// VolumeAttachment objects are non-namespaced.
+// VolumeAttachment objects are non-namespaced. Captures the intent to attach or detach the specified volume to/from the specified node.
```

Comment:
nit: please the break the line after "or", others LGTM

Reply:
@draveness fixed!